data/human/news.txt

```diff
@@ -2150,3 +2150,313 @@ news "graffiti on spaceport wall"
+		"the tyranny of a man name Bailey, until he was overthrown"
+		"CCOR, praising their accomplishments"
+		"resistance, opposition to the current planetary administration"
+
+phrase "Worker's Demands"
+	word
+		"safety regulations"
+		"increased pay"
+		"restoration of the CCOR"
+		"justice against planetary security brutality"
+		"immediate repositioning of the spaceport"
+
+news "Discontent"
+	location
+		planet Bailey
+	name
+		"Conflict"
+	message
+		word
+			"Allegations that planetary authorities will no longer move the spaceport has provoked "
+		word
+			"large scale "
+			"mass "
+			"moderate "
+			"scattered "
+		word
+			"riots "
+			"protests %"
+		word
+			"the rioters are "
+		word
+			"singing, %{CCOR Song Description}. "
+			"chanting, a list of their demands including ${Worker's Demands}. "
+			"sizing up the planetary security forces. "
+		word
+			"Security eventually steps in, "
+		word
+			"however they're pushed back. "
+			"but they're routed. "
+			"and they disperse the demonstrators. "
+			"and the demonstrators scatter but don't appear dissuaded."
+		replace
+			"% the rioters" "the protesters"
+			"%" ""
+
+news "Clandestine"
+	location
+		planet Alvorada
+name
+	"Observation"
+message
+	word
+		"You can't escape the feeling of being watched. It"
+	word
+		"'s probably the security cameras; most of them are still running."
+		" feels like there are eyes around every corner."
+		"'s the pirates; a small number of them seem to be observing you."
+	word
+		"After some time of this "
+	word
+		"a silent alarm tells you that your should check on your ship. "
+		"it may be wise make sure no-one's tampering with your ship. "
+	word
+		"There's "
+	word
+		"a lone pirate prodding the ship with rudimentary tools. "
+		"a few pirates watching but not apparently touching it. "
+		"noone othere, the area around your ship is almost suspiciously devioid of people. "
+	word
+		"Whatever's going on your ship appears unscathed."
+
+news "Microgravity"
+	location
+		planet Mordente-Bridi
+name
+	"Repairs"
+message
+	word
+		"Watching the micro-gravity section of the spaceport from the bar, you "
+	word
+		"observe "
+		"notice "
+		"spot "
+		"spy "
+	word
+		"a Bastion "
+		"an Aerie "
+		"a Falcon "
+		"a Leviathan "
+	word
+		"returning from a raid, "
+	word
+		"potmarked with laser burns but no apparent serious damage. "
+		"battered with thousands of small perferations. Somebody's turned a mainstay of the pirate arsenal, gatling guns, back on them. "
+		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missle. "
+	word
+		"As it comes to a stop in the maitenance bay engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"The work won't take the shuttles long."
+		"They'll be working for the rest of the shift on this one."
+
+news "Symbol War"
```

Comment:
Also, this seems to be more of a mission than a spaceport news entry in its layout.